.githooks/pythonScripts/check_api_key.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/local/bin/python3
+
+'''
+Name:        check_api_key.py
+
+Purpose:
+This script parses files staged for commit to ensure
+that they do not contain an API key.
+
+Notes:
+
+    Unity: apiKey:  APIKeyValue
+
+    Block Commit:
+        * Anything that contains AAPK
+        * apiKey: APIKeyString - API key is set to any none empty value
+        * apiKey: (1"asdf"df*) - API key is invalid, though may still contain sensitive information
+
+    Allow Commit:
+        * apiKey: - empty value
+        * apiKey: "YOUR_API_KEY" - Citra requested snippet
+        * apiKey: not found in any staged files
+'''
+import re
+import argparse
+
+#-------------------------------------------------------------------------------
+# Global Variables
+#-------------------------------------------------------------------------------
+
+content = []
+
+unity_apiKey_argument_regex = r"apiKey[\s]*\:[\s]*([\sa-zA-Z0-9_\"\'\-]*)"
```

Comment:
This doesn't actually check for keys, you should be checking for the AAPK string

Reply:
Yup! That's checked on line 65